.gitignore

```diff
@@ -6,3 +6,4 @@ venv
 *.pyc
 test/sagemaker_tests/huggingface/inference/resources/**/*[.json,.h5,.bin,.txt]
 build/
+buildfiles/
```

Comment:
Why is this necessary?

Reply:
This expression will be removed. Needed it to make sure that I don't check in the content of the buildfiles along side with other changes by mistake.